### ISSUE TITLE
CTRL+L exception

### ISSUE THREAD

**Issue by hholst80** [PowerShell/PSReadLine#1105]
```
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 64 Keys:
 s s h Space h o l s t @ i t Enter
 ` Ctrl+l a s d f a s d Ctrl+c Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l j k j k j k j k j k j Backspace Ctrl+l Ctrl+l Ctrl+l Ctrl+l Ctrl+l

Exception:
System.IO.IOException: The parameter is incorrect.

   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.Console.SetWindowPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

Environment data
----------------

```
PS version: 5.1.18362.145
PSReadline version: 2.0.0-beta2
os: 10.0.18362.1 (WinBuild.160101.0800)
PS file version: 10.0.18362.1 (WinBuild.160101.0800)
HostName: ConsoleHost
BufferWidth: 157
BufferHeight: 58
```

**Comment by msftrncs** [comment]
@hholst80, give 2.0.0-beta5 a try, from PowerShellGallery .  I think it will resolve this, if not, please report back.

**Comment by daxian-dbw** [comment]
@hholst80 I will close this issue as it's likely fixed in 2.0.0-beta5. Please re-open if not, and provide the environment data (the issue template has the script for you to collect the environment data).
